actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,7 +45,11 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller = controller.to_s =~ %r{\arails/(info|mailers|welcome)}
+        internal_asset = Rails.application.config.respond_to?(:assets) &&
```

Comment:
Why config is not responding to `assets` anymore? I could not find where we remove it.